deployments/scroll-goerli/usdc/migrations/1694594095_configurate_and_ens.ts

```diff
@@ -0,0 +1,232 @@
+      {
+        contract: scrollMessenger,
+        signature: 'sendMessage(address,uint256,bytes,uint256)',
+        args: [bridgeReceiver.address, 0, l2ProposalData, 600_000]
+      },
+
+      // 2. Approve Goerli's StandardERC20Gateway to take Timelock's COMP (for bridging)
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [scrollL1StandardERC20GatewayAddress, COMPAmountToBridge]
+      },
+      // 3. Bridge COMP from Goerli to Scroll Alpha Comet using L1GatewayRouter
+      {
+        contract: scrollL1TokenBridge,
+        signature: 'depositERC20(address,address,uint256,uint256)',
+        args: [COMP.address, rewards.address, COMPAmountToBridge, 300_000]
+      },
+      // 4. Update the list of official markets
+      {
+        target: ENSResolverAddress,
+        signature: 'setText(bytes32,string,string)',
+        calldata: ethers.utils.defaultAbiCoder.encode(
+          ['bytes32', 'string', 'string'],
+          [subdomainHash, ENSTextRecordKey, JSON.stringify(officialMarketsJSON)]
+        )
+      }
+    ];
+
+    const description =
+      '# Configurate Scroll Alpha cUSDCv3 market, set reward config, bridge over USDC and COMP, and update ENS text record.';
+    const txn = await govDeploymentManager.retry(async () =>
+      trace(await governor.propose(...(await proposal(goerliActions, description))))
+    );
+
+    const event = txn.events.find(event => event.event === 'ProposalCreated');
+    const [proposalId] = event.args;
+
+    trace(`Created proposal ${proposalId}.`);
+  },
+
+  async enacted(deploymentManager: DeploymentManager): Promise<boolean> {
+    return true;
+  },
+
+  async verify(
+    deploymentManager: DeploymentManager,
+    govDeploymentManager: DeploymentManager,
+    preMigrationBlockNumber: number
+  ) {
+    const ethers = deploymentManager.hre.ethers;
+    await deploymentManager.spider(); // We spider here to pull in Scroll COMP now that reward config has been set
+
+    const { comet, rewards, COMP, USDC } = await deploymentManager.getContracts();
+
+    // 1.
+    const stateChanges = await diffState(comet, getCometConfig, preMigrationBlockNumber);
+    expect(stateChanges).to.deep.equal({
+      baseTrackingSupplySpeed: exp(34.74 / 86400, 15, 18),
+      baseTrackingBorrowSpeed: exp(34.74 / 86400, 15, 18),
+      baseMinForRewards: exp(1000, 6),
+      WETH: {
+        borrowCollateralFactor: exp(0.775, 18),
+        liquidationFactor: exp(0.95, 18),
+        supplyCap: exp(1000, 18)
+      },
+    });
+
+    const config = await rewards.rewardConfig(comet.address);
+    expect(config.token).to.be.equal(COMP.address);
+    expect(config.rescaleFactor).to.be.equal(exp(1, 12));
+    expect(config.shouldUpscale).to.be.equal(true);
+
+    // 2. & 3.
+    expect(await COMP.balanceOf(rewards.address)).to.be.equal(exp(1_000, 18));
+
+    // 4 & 5,
+    expect(await USDC.balanceOf(rewards.address)).to.be.equal(exp(100_000, 6));
```

Comment:
This should be removed. If you run scenarios locally, I believe it'll fail here.